handbook/engineering/core-services/index.md

```diff
@@ -1,11 +1,66 @@
 # Code services team
 
-[Roadmap](https://docs.google.com/document/d/1cBsE9801DcBF9chZyMnxRdolqM_1c2pPyGQz15QAvYI/edit#heading=h.fv5i7qi85bru)
-
 ## Vision statement
 
-Sourcegraph operations are fast (e.g. search), instances scale to 80k repositories, private code is secure and respects ACLs for every authorization provider, and the product settings are easy to set up and understand for our largest customers. Tech stack: Go, Postgres, Redis, Docker, Kubernetes, and others.
+Sourcegraph operations are fast (e.g. search), instances scale to 80k repositories, private code is secure and respects ACLs for every authorization provider, and the product settings are easy to set up and understand for our largest customers.
 
 ## Tech stack
 
-Go, Postgres, Redis, Docker, Kubernetes.
\ No newline at end of file
+Go, Postgres, Redis, Docker, Kubernetes.
+
+## Resources
+
+- [Roadmap](https://docs.google.com/document/d/1cBsE9801DcBF9chZyMnxRdolqM_1c2pPyGQz15QAvYI/edit#heading=h.fv5i7qi85bru)
+- [Tracking Issues](https://github.com/sourcegraph/sourcegraph/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3Ateam%2Fcore-services+label%3Atracking)
+
+## Processes
+
+### Weekly check-ins
+
+Everyone writes a weekly check-in at the end of each Friday in the active tracking issue of the on-going iteration. These check-ins describe the work done in the previous week as well as what's to come in the next week.
+
+This process helps each team mate reflect and plan each week proactively and enables other parts of the organization to have visibility into on-going progress.
+
+Here's an example of a [good weekly check-in](https://github.com/sourcegraph/sourcegraph/issues/7190#issuecomment-573564817).
+
+### Monthly planning
+
+Planning happens monthly in the last week of the current iteration to match [Sourcegraph's monthly release cycle](../releases/index.md).
+
+The team manager and a senior team mate set-up a block of time together to triage all open issues [labeled with **team/core-services**](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aopen+is%3Aissue+label%3Ateam%2Fcore-services).
+
+Each issue is:
+
+- Closed if appropriate.
+- Commented on if it's unclear what next steps are.
+- Assigned to the up-coming milestone if deemed a priority.
+- Left in the backlog otherwise.
```

Comment:
Is this valid: break up the issue if it helps with estimation and fit into planning, i.e. the idea in the original issue was too broad.

Reply:
Yes, good one! I'll add this.

Reply:
Actually, we already talk about that in the context of roadmap items and spikes below, so I think it'd be redundant to have it here as well.